diff --git a/src/Controller/SongController.php b/src/Controller/SongController.php
index f278499..0192d82 100644
--- a/src/Controller/SongController.php
+++ b/src/Controller/SongController.php
@@ -2,6 +2,7 @@
 
 namespace App\Controller;
 
+use Psr\Log\LoggerInterface;
 use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
 use Symfony\Component\Routing\Annotation\Route;
 
@@ -20,9 +21,12 @@ class SongController extends AbstractController
     /**
      * @Route("/another-song")
      */
-    public function writeAnotherSong()
+    public function writeAnotherSong(LoggerInterface $logger)
     {
         $song = 'Back-road, boot-scooting, honkey-tonkin CMS';
+        $logger->info('Time to sing!', [
+            'song' => $song,
+        ]);
 
         return $this->render('song/anotherSong.html.twig', [
             'song' => $song,
