diff --git a/config/routes.yaml b/config/routes.yaml
index c3283aa..dfbaf3c 100644
--- a/config/routes.yaml
+++ b/config/routes.yaml
@@ -1,3 +1,3 @@
-#index:
-#    path: /
-#    controller: App\Controller\DefaultController::index
+song_api_write_song:
+    path: /api/songs
+    controller: App\Controller\SongController::apiWriteSong
diff --git a/src/Controller/SongController.php b/src/Controller/SongController.php
new file mode 100644
index 0000000..621c18e
--- /dev/null
+++ b/src/Controller/SongController.php
@@ -0,0 +1,15 @@
+<?php
+
+namespace App\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+
+class SongController extends AbstractController
+{
+    public function apiWriteSong()
+    {
+        return $this->json([
+            'I rode my truck, through some mud',
+        ]);
+    }
+}
