diff --git a/config/routes.yaml b/config/routes.yaml
index dfbaf3c..5492f57 100644
--- a/config/routes.yaml
+++ b/config/routes.yaml
@@ -1,3 +1,3 @@
-song_api_write_song:
-    path: /api/songs
-    controller: App\Controller\SongController::apiWriteSong
+#song_api_write_song:
+#    path: /api/songs
+#    controller: App\Controller\SongController::apiWriteSong
diff --git a/src/Controller/SongController.php b/src/Controller/SongController.php
index 621c18e..4b3acfd 100644
--- a/src/Controller/SongController.php
+++ b/src/Controller/SongController.php
@@ -3,9 +3,13 @@
 namespace App\Controller;
 
 use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\Routing\Annotation\Route;
 
 class SongController extends AbstractController
 {
+    /**
+     * @Route("/api/songs")
+     */
     public function apiWriteSong()
     {
         return $this->json([
